src/ansys/dpf/core/server_context.py

```diff
@@ -38,6 +40,111 @@ def same_licensing_context(first, second):
         return True
 
 
+class LicenseContextManager:
+    """Can be optionally used to pre checkout a licence before using licensed DPF Operators.
+    Improves performances if many Operators requiring licensing are used afterwards.
+    It can also be used to force checkout before running a script when few
+    ANSYS license increments are available.
+    Check in the license when the object is deleted.
```

Comment:
```suggestion
    """Can optionally be used to check out a license before using licensed DPF Operators.
    Improves performance if you are using multiple Operators that require licensing.
    It can also be used to force checkout before running a script when few
    Ansys license increments are available.
    The license is checked in when the the object is deleted.
```